system/ee/legacy/libraries/Core.php

```diff
@@ -39,7 +39,7 @@ public function bootstrap()
         }
 
         // Set a liberal script execution time limit, making it shorter for front-end requests than CI's default
-        if (function_exists("set_time_limit") == true) {
+        if (function_exists("set_time_limit") == true && php_sapi_name() !== 'cli') {
```

Comment:
Just to confirm... I assume this then defaults to no limit?  (Any chance you ran a quick dump to confirm or anything along those lines?  If not I can run a quick test.)

Tagging @mithra62

Reply:
Yes, no limit for CLI requests is what this does. I did test it to ensure the code for setting the memory limit doesn't execute on my local with the PR branch. I'm not following "a quick dump to confirm" though so you might wanna test just to be safe.

Reply:
Yeah, that's what I was wondering... basically just confirmation that it took effect as intended (time limit wasn't being set somewhere else or something along those lines) 

Right on!

Reply:
You bring up a good point. I've been kicking around on whether to submit a ticket for this, but EE does set the execution time in numerous places across the code. 

![image](https://user-images.githubusercontent.com/1008036/144937721-0733df65-3053-4dc2-af26-508b4f18fcde.png)

Only one of those is engaged by default when used on Commands though abstracting that may help solve future Command based tasks.